css-view-transitions-1/Overview.bs

```diff
@@ -705,31 +768,22 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	}
 	```
 
-### <dfn>::view-transition-old( <<pt-name-selector>> )</dfn> ### {#::view-transition-old}
-
-	<div class=note>
-
-		This element is a replaced element that produced the visual representation of the “old” state taken from user-agent provided snapshots.
+### View Transition Old State Image: the ''::view-transition-old()'' pseudo-element ### {#::view-transition-old}
 
-		This is only ever a [=tree/child=] of a ''::view-transition-image-pair()'',
-		never has any [=tree/children=],
-		and is omitted if there's no “old” state to represent.
+	The <dfn>::view-transition-old()</dfn> [=pseudo-element=]
+	is an empty [=named view transition pseudo-element=]
```

Comment:
Does "empty" here imply that it has no children?

Reply:
yes, that was the intention